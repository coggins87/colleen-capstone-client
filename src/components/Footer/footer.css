.footer {
  color: rgb(201, 197, 192);
  display: flex;
  align-content: space-between;
  width: 100vw;
  font-size: 0.6em;
}
.instructions {
  display: inherit;
  margin: 0 auto;
}
.external_links {
  display: inherit;
  flex-direction: column;
  text-align: right;
  color: white;
  margin: auto;
}

.external_links > p > a {
  margin: 0 auto;
  flex-wrap: nowrap;
  color: rgb(201, 197, 192);
  text-decoration: none;
}
.external_links > p > a:hover {
  color: white;
}
