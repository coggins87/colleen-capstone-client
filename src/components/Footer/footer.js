import React from "react";
import "./footer.css";
export default function Footer() {
  return (
    <div className="footer">
      <div className="instructions">
        <p>
          To log in as test user: <br />
          Username: Test<br />
          Password: Test123!
        </p>
      </div>
      <div className="external_links">
        <p>
          <a href="https://github.com/thinkful-ei-emu/colleen-capstone-client">Project Code: Client</a>
        </p>
        <p>
          <a href="https://github.com/thinkful-ei-emu/amrap-api-colleen">Project Code: Server</a>
        </p>
        <p>
          <a href="https://thinkful-ei-emu.github.io/portfolio-colleen/">My Portfolio</a>
        </p>
      </div>
    </div>
  );
}
