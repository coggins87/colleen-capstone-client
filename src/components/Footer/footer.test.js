import React from 'react';
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import Footer from './footer'

describe ('Footer', ()=>{
  it('renders a footer by default', ()=>{
    const wrapper = shallow(<Footer />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})