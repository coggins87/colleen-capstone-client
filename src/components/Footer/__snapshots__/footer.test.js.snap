// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders a footer by default 1`] = `
<div
  className="footer"
>
  <div
    className="instructions"
  >
    <p>
      To log in as test user: 
      <br />
      Username: Test
      <br />
      Password: Test123!
    </p>
  </div>
  <div
    className="external_links"
  >
    <p>
      <a
        href="https://github.com/thinkful-ei-emu/colleen-capstone-client"
      >
        Project Code: Client
      </a>
    </p>
    <p>
      <a
        href="https://github.com/thinkful-ei-emu/amrap-api-colleen"
      >
        Project Code: Server
      </a>
    </p>
    <p>
      <a
        href="https://thinkful-ei-emu.github.io/portfolio-colleen/"
      >
        My Portfolio
      </a>
    </p>
  </div>
</div>
`;
